src/test/scala/com/github/music/of/the/ainur/almaren/Test.scala

```diff
@@ -251,57 +251,6 @@ class Test extends FunSuite with BeforeAndAfter {
     }
   }
 
-  def testSourceSql(tableName: String): DataFrame = {
```

Comment:
Why did you delete those test cases ?

Reply:
In previous PR these def got repeated, removed duplicated ones and tested